router/deployment.yaml

```diff
@@ -43,20 +43,6 @@ spec:
 ---
 apiVersion: v1
 kind: Service
-metadata:
-  name: batch
```

Comment:
One is for the batch front end and the other is for the driver.

Reply:
The driver one is the next one down, it's app is `batch-driver`. The front end one is duplicated. The preserved one is the entry immediately before this deleted entry.